### PR TITLE
feat: localization

### PR DESCRIPTION
**Pull request by IanKrieger**
Localization string extraction related to: https://github.com/brave/roadmap/issues/1115

### CODE REVIEW

File: src/auth/hooks/mutations/useSignOut.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { clearCredentials } from "auth/lib";
 
 interface Options {
   onSuccess?: () => void;
-  onError?: (msg: string) => void;
+  onError?: (mmessagesg: string) => void;
```
Comment (tackley) [inline src/auth/hooks/mutations/useSignOut.ts:7]: nit: sp?

File: tsconfig.json [hunk 2]
```diff
@@ -19,4 +19,5 @@
     "baseUrl": "src",
   },
   "include": ["src"],
+  "exclude": ["*.test.ts"],
```
Comment (tackley) [inline tsconfig.json:22]: why do we want to exclude tests from typescript compilation?

Comment (IanKrieger) [reply tsconfig.json:22]: I was trying to exclude from eslint temporarily. Wrong place, I realized but did not revert.

File: lingui.config.js
[the remark this answers is not in the document]
Comment (IanKrieger) [reply lingui.config.js:1]: Hadn't looked at this, will investigate!

[the remark this answers is not in the document]
Comment (tackley) [reply lingui.config.js:1]: let's consider as a follow up ;)

**Review by tackley — APPROVED** [review]
what a lot of strings!! but looks like a great implementation to me...

nit: there's a lot of gunk coming out to the browser console on startup, the 401 is unavoidable but the rest is messy. Can we clean this up?

Screenshot 2024-02-15 at 10 23 38

**Pull request merged**